php_fpm/datadog_checks/php_fpm/data/conf.yaml.example

```diff
@@ -52,71 +69,31 @@ instances:
     #
     ping_url: http://localhost/ping
 
-    ## @param use_fastcgi - boolean - required - default: false
-    ## Communicate directly with PHP-FPM using FastCGI
-    #
-    use_fastcgi: false
-
     ## @param ping_reply - string - required
     ## Set the expected reply to the ping.
     #
     ping_reply: pong
 
+    ## @param use_fastcgi - boolean - optional - default: false
```

Comment:
There is actually a default value to `False` https://github.com/DataDog/integrations-core/blob/458a98df150ca50b5f2eaab0bc276b8272fa2d7e/php_fpm/datadog_checks/php_fpm/php_fpm.py#L80